src/lib/Microsoft.Health.Fhir.Ingest/Service/MeasurementFhirImportService.cs

```diff
@@ -87,6 +108,31 @@ private static async Task<IEnumerable<IMeasurementGroup>> ParseAsync(Stream data
             return measurementGroups;
         }
 
+        private static IEnumerable<IMeasurementGroup> ParseEventData(IEnumerable<IEventMessage> data)
+        {
+            // Deserialize events into measurements and then group according to the device, type, and other factors
+            var body = data.First().Body.ToArray();
+            var text = System.Text.Encoding.Default.GetString(body);
+            var measurement = JsonConvert.DeserializeObject<Measurement>(text);

```

Comment:
Maybe I'm missing something, but it looks like measurement isn't used? what is the purpose of deserializing the first message?

Reply:
It should be removed. I think I may have been logging the event for debugging but didn't remove these lines when I was done.